File: csharp/PrimeNumbers.cs
using System;

class PrimeNumbers {
    static void Main(string[] args) {
        for (int i = 2; i <= 100; i++) {
            if (IsPrime(i)) {
                Console.Write(i + " ");
            }
        }
    }

    static bool IsPrime(int num) {
        if (num <= 1) return false;
        for (int i = 2; i <= Math.Sqrt(num); i++) {
            if (num % i == 0) return false;
        }
        return true;
    }
}

File: JavaScript/prime_numbers.js
function isPrime(num) {
    if (num <= 1) return false;
    for (let i = 2; i <= Math.sqrt(num); i++) {
        if (num % i === 0) return false;
    }
    return true;
}

let primes = [];
for (let i = 2; i <= 100; i++) {
    if (isPrime(i)) primes.push(i);
}

console.log(primes);
